### ISSUE TITLE
Demo link is broken

### ISSUE THREAD

**Issue by arthurtalkgoal** [konung/comfy_imprint#10]
http://www.gorbikoff.com/contact-us

Hi, the above link is broken page.

**Comment by konung** [comment]
Yes sorry. I was playing with my VM - upgrading and testing other stuff.
I'm not maintaining the my blog right now, so I didn't notice it wasn't working. The plugin should still work for older comfy, that I designed it for, but I haven't tried new CMS  - so I can't make any promises.
